code/ATMOSPHERICS/components/binary_devices/passive_gate.dm

```diff
@@ -155,3 +155,14 @@
 /obj/machinery/atmospherics/binary/passive_gate/power_change()
 	..()
 	update_icon()
+
+/obj/machinery/atmospherics/binary/passive_gate/npc_tamper_act(mob/living/L)
+	if(prob(50)) //Turn on/off
+		on = !on
+		investigation_log(I_ATMOS,"was turned [on ? "on" : "off"] by [L] ([key_name(L)])")
+	else //Change pressure
+		src.target_pressure = rand(0, 4500)
```

Comment:
So this is weird.

`Topic()` code implies the limit here is `4500`, while `receive_signal` implies `50 * ONE_ATMOSPHERE`.

Move it to a constant either way.